lib/galaxy/model/store/ro_crate_utils.py

```diff
@@ -222,6 +225,162 @@ def _add_workflows(self, crate: ROCrate):
             crate.mainEntity["name"] = self.workflow.name
             crate.mainEntity["subjectOf"] = cwl_wf
 
+            # Adding multiple creators if available
+            if self.workflow.creator_metadata:
+                for creator_data in self.workflow.creator_metadata:
+                    if creator_data.get("class") == "Person":
+                        # Create the person entity
+                        creator_entity = crate.add(
+                            ContextEntity(
+                                crate,
+                                creator_data.get("identifier", ""),  # Default to empty string if identifier is missing
+                                properties={
+                                    "@type": "Person",
+                                    "name": creator_data.get("name", ""),  # Default to empty string if name is missing
+                                    "orcid": creator_data.get(
```

Comment:
Just noticed - this property should be called `identifier`, not `orcid` (there is not a specific `orcid` term in RO-Crate)
```suggestion
                                    "identifier": creator_data.get(
```